.github/workflows/transifex_merge_translations.yml

```diff
@@ -0,0 +1,37 @@
+# Since transifex-synchronization-source is just a staging branch to control
+# updates to Transifex, we need to force merge the translations to master,
+# excluding the English source files
+name: Translations merge to master
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - transifex-synchronization-source
+
+jobs:
+  merge-translations:
+    if: github.event.pull_request.user.login == 'transifex-integration[bot]' && github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v4
+        with:
+          ref: master
+
+      - name: Setup Git User
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+      - name: Force merge translation directories (excluding English)
+        run: |
+          git fetch --unshallow origin transifex-synchronization-source
```

Comment:
Why `--unshallow` ? It's quite expensive and it's not clear to my why it's needed

Reply:
Leftover from when I was trying to troubleshoot why it was failing, I will remove and test.